### ISSUE TITLE
NTR: Pyrroline-5-carboxylate high in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8289]
New term request Pyrroline-5-carboxylate high in blood (elthat_111112203807): P5C; corrected for time of day

**Comment by pnrobinson** [comment]
@MickeySegal
is this referring to 
[1-pyrroline-5-carboxylate](https://www.ebi.ac.uk/ols4/ontologies/chebi/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FCHEBI_15893) ?

**Comment by MickeySegal** [comment]
https://omim.org/clinicalSynopsis/239510 lists "Delta-1-pyrroline-5-carboxylate (P5C) levels increased in plasma".  So maybe even with the delta too, and also for "Delta-1-pyrroline-5-carboxylate dehydrogenase activity" discussed in https://github.com/obophenotype/human-phenotype-ontology/issues/8290

**Comment by pnrobinson** [comment]
@MickeySegal 
according to metabolome they re all synonyms.

Adding new term: 
- ID: HP:6000694
- label: Elevated circulating 1-pyrroline-5-carboxylic acid acid concentration
- parents: [Abnormal circulating monocarboxylic acid concentration (HP:0010996)](https://hpo.jax.org/app/browse/term/HP:0010996)
- pmid: PMID:9700195
- def: The concentration of delta-1-pyrroline-5-carboxylate in the blood circulation is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8289
- synonyms: Elevated circulating delta-1-pyrroline-5-carboxylate concentration(NONE)
